source/agora/consensus/Genesis.d

```diff
@@ -80,27 +133,33 @@ private immutable ubyte[] GenesisAddressUbyte =
 
 unittest
 {
-    assert(GenesisOutputAddress.toString()
+    assert(UnitTestGenesisOutputAddress.toString()
            == `GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ`);
 }
 
+unittest
+{
+    testSymmetry(UnitTestGenesisTransaction);
+    testSymmetry(UnitTestGenesisBlock);
+}
 
 version (unittest)
 {
     /***************************************************************************
 
-        Get the key-pair which can spend the UTXO in the genesis transaction.
+    Get the key-pair which can spend the UTXO in the unittests
+    genesis transaction.
```

Comment:
This `+unittests` wording change should be moved to a previous commit.

There will be no key-pair for the coinnet transactions, because only the public key will be visible and not the private key.

Or is something else not clear?

Reply:
Sorry, my incredulity was towards the broken indentation ;)